### PR TITLE
create hubspot blog post on release

### PR DESCRIPTION
**Pull request by garyverhaegen-da**

### CODE REVIEW

**Comment by hurryabit** [comment]
Could you please add a short description what this PR does? I suppose it's publishing the release notes on HubSpot?

**Comment by garyverhaegen-da** [comment]
> Could you please add a short description what this PR does? I suppose it's publishing the release notes on HubSpot?

It's creating a blop post on hubspot when we make a release. I figured the PR title was enough.

Note that I've already set up the `HUBSPOT_TOKEN` env var in Azure, and the added bash code is lifted directly from a bash script that *Works On My Machine*™®, so this will obviously work as intended.

Running the script on my machine, setting `LATEST` to `v0.13.14`, has produced [this post](https://blog.daml.com/release-notes/0.13.14).

File: azure-cron.yml [hunk 1]
```diff
@@ -101,10 +103,34 @@ jobs:
           aws cloudfront create-invalidation \
                          --distribution-id E1U753I56ERH55 \
                          --paths '/*'
+          echo "Publishing $LATEST to Hubspot..."
+          DESCRIPTION_MD="$(cat $GH_RESP | jq -r '. | map(select(.tag_name="'$LATEST'"))[0].body')"
+          DESCRIPTION=$(curl -s -XPOST 'https://api.github.com/markdown/raw' -H 'Content-Type: text/plain' -d"$DESCRIPTION_MD" | jq -sR .)
+          DATE="$(date -d $(cat $GH_RESP | jq -r '. | map(select(.tag_name="'$LATEST'"))[0].published_at') +%s)000"
+          BLOG=11411412838
```
Comment (hurryabit) [inline azure-cron.yml]: Where is this magic number coming from? Is it the ID of our blog? Same for `blog_author_id` below/

Comment (garyverhaegen-da) [reply azure-cron.yml]: Yes, id of our blog and blog author, respectively. I've added a line of comment to clarify.

**Comment by garyverhaegen-da** [comment]
> Nice. Thanks a lot.
> 
> Is there a chance we can change the title (as in the very first line of the post) from
> 
> > xX.Y.Z
> 
> to something like
> 
> > Release of DAML SDK vX.Y.Z

Done.

**Pull request merged**